docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Keyacrd? (have no idea how to call it)
+version: 0.0.0
+status: Draft
```

Comment:
Missing id, version 0.1

Reply:
should I just use some random number as id or?

Reply:
next in succession based on existing ones and open PRs